cmd/aad-cli/cli/testdata/golden/TestUserShellCompletion/get_attributes_for_overridden_user

```diff
@@ -0,0 +1,9 @@
+login
+password
+uid
+gid
+gecos
+home
+shell
+last_online_auth
+:4
```

Comment:
I have one very stupid question: where do you set the return code in stdout? I don’t find it in RunApp() testutils helper, nor in the cmd.Run() command, so I’m puzzled :)